.github/ISSUE_TEMPLATE/support-needed.md

```diff
@@ -2,8 +2,30 @@
 name: Support Needed
```

Comment:
we should probably delete this template
discussions should be moved to "Discussions"

Reply:
I think about that too, but I also think `support-needed` is something between 'bug' and 'Discussion''.
In our flow, we can route `support-needed` to `Discussion` if necessary.